src/components/FilterPanel/LEGACY_FilterPanel/FilterSelector/FilterSelector.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file Filter selector.
+ * @copyright IBM Security 2019
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { getComponentNamespace } from '../../../../globals/namespace';
+
+import { filterFiltersPropTypes } from '../FilterPanelUtilities';
+import FilterCheckbox from '../../FilterCheckbox';
+
+export const namespace = getComponentNamespace('filter-selector');
+
+const FilterSelector = ({ filter, onChange }) => (
```

Comment:
Two things --

- I was wondering why `FilterCheckbox` has to be wrapped like this, as it looks like just an `onChange` is getting added?

- Also, could you add an `...other` if you want to keep this?

Reply:
1. This component is a legacy component and is expected to work with a filter object that includes specific properties likes `enabled`, `id`, `count`, and `name` among others. To reduce the amount of duplicate code, I am reusing the new `FilterCheckbox` component to achieve the visuals, while maintaining the legacy data requirements.

2. Same comment about not needing to add `...other` to legacy components.